ignite/services/network/networktypes/profile.go

```diff
@@ -96,7 +96,7 @@ type (
 	// Profile represents the address profile on SPN
 	Profile struct {
 		Address         string               `json:"Address"`
-		CampaignID      uint64               `json:"CampaignID,omitempty"`
+		CampaignID      uint64               `json:"ProjectID,omitempty"`
```

Comment:
```suggestion
		CampaignID      uint64               `json:"CampaignID,omitempty"`
```
Should be consistent with the name of the field